src/pages/Lending/YourLoans.tsx

```diff
@@ -35,6 +36,16 @@ const LendingRowGrid = ({ asset, accountId, onPoolClick }: LendingRowGridProps)
       accountId,
     })
 
+  const useRepaymentLockDataArgs = useMemo(
+    () => ({
+      assetId: asset.assetId,
+      accountId,
+    }),
```

Comment:
get dem lines
```suggestion
    () => ({ assetId: asset.assetId, accountId }),
```

Reply:
eslint is happy so I'm happy b4055aa5aa79bd13394411d42cc37f5c985f34cf